src/main/java/seedu/address/logic/commands/ManageVolunteerCommand.java

```diff
@@ -0,0 +1,69 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_VOLUNTEERS;
+
+import java.util.List;
+
+import seedu.address.commons.core.EventsCenter;
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.events.ui.ContextChangeEvent;
+import seedu.address.commons.events.ui.RecordChangeEvent;
+import seedu.address.logic.CommandHistory;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.event.Event;
+import seedu.address.model.record.RecordContainsEventIdPredicate;
+
+/**
+ * Selects a person identified using it's displayed index from the address book.
+ */
+public class ManageVolunteerCommand extends Command {
```

Comment:
Volunteers do not need a manage command. Manage is just used in the event's context to switch to the record context.